src/OpalCompiler-Core/CompilationContext.class.st

```diff
@@ -466,7 +481,8 @@ CompilationContext >> getClass [
 CompilationContext >> initialize [
 	options := Set new.
 	astTransformPlugins := OrderedCollection withAll: self class defaultTransformationPlugins.
-	semanticScope := OCMethodSemanticScope targetingClass: nil class
+	semanticScope := OCMethodSemanticScope targetingClass: nil class.
+	astParseTransformPlugins := OrderedCollection new: 2
```

Comment:
Is it really needed to allocate it with 2?

Reply:
it is a size optimization: most of the time there will not be parse plugin so allocate 10 slots for it seemed a lot, but anyway, that's already addressed.